bluesky_widgets/qt/_tests/test_figures.py

```diff
@@ -56,6 +56,25 @@ def test_figure_title_syncing(qtbot):
     assert view.figure._suptitle.get_text() == expected
 
 
+def test_short_title_syncing(qtbot):
+    model = func(run)
+    figures = FigureSpecList([model])
+    view = QtFigures(figures)
+    actual_title = view.figures[next(iter(view.figures))].figure._suptitle.get_text()
```

Comment:
The `next(iter(...))` trick is useful for collections that are not indexable, but in this case I think we just have a `list`, so the simpler `view.figures[0]` will work.

Reply:
As you reminded me offline, this is a `DictView`, not a `list`. The dict is keyed on `FigureSpec.uuid`, so the way to access an item here is `view.figures[model.uuid]`.